backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/query/impl/ElasticsearchSearchQueryBuilder.java

```diff
@@ -125,8 +125,8 @@ public void failAfter(long timeout, TimeUnit timeUnit) {
 	}
 
 	@Override
-	public void totalHitCountMinimum(int totalHitCountMinimum) {
-		this.totalHitCountMinimum = totalHitCountMinimum;
+	public void totalHitsThreshold(int totalHitsThreshold) {
```

Comment:
`totalHitsThreshold` is the Lucene name. but I agree with you, it is more `totalHitCountThreshold`, since the target of the threshold is the count and not the hits.